aten/src/ATen/native/vulkan/gen_spv.py

```diff
@@ -0,0 +1,142 @@
+
```

Comment:
Can you put a `#!/usr/bin/env python3` here just in case someone tries to run this directly?